frontend/components/text2text/results/Text2TextList.vue

```diff
@@ -16,115 +16,98 @@
   -->
 
 <template>
-  <div>
+  <div
+    :class="[
+      'content',
+      'content--separator',
+      !annotationEnabled
+        ? 'content--exploration-mode'
+        : 'content--annotation-mode',
+    ]"
```

Comment:
separate the binded classes and the other classes =>
`
class="content content--separator"
:class="[
      !annotationEnabled
        ? 'content--exploration-mode'
        : 'content--annotation-mode',
    ]"
`